pkg/Microsoft.Private.CoreFx.NETCoreApp/netcoreapp.rids.props

```diff
@@ -21,6 +21,7 @@
     <OfficialBuildRID Include="win-x86">
       <Platform>x86</Platform>
     </OfficialBuildRID>
+    <OfficialBuildRID Include="@(OfficialBuildRID->'%(Identity)-aot')" />
```

Comment:
Note that this will double the number of official builds.

Reply:
As we have discussed offline some time ago, I think it would be better to finish refactoring the CoreRT/AOT CoreLib so that we can have one CoreFX build that works for both non-AOT and AOT. It would avoid doubling of the official build flavors.

Reply:
I checked with @ericstj and this file doesn't actually control official build. I didn't add an official build in this change; I'll do that and CI separately. When I do, I'll just add one more build.

Reply:
> I think it would be better to finish refactoring the CoreRT/AOT CoreLib so that we can have one CoreFX build that works for both non-AOT and AOT.

I think that's a great goal, but when I tried to experiment with using the existing netcoreapp, there were two issues:
1. There are still lots of differences that would need refactoring.
2. Testing needs to work differently (running ILC, etc).

By making this change, we can get testing for the runtime and compiler with this framework as well as having all of the remaining differences effectively flagged by the build variables (since this separates aot and uap). If the refactoring is completed, we can look at removing the TargetGroup.

Reply:
I'm ok with this change.  It does mean that the runtime.json inside MS.Private.CoreFx.NetCoreApp will have AOT rids (which may be dangling refs if we don't have official builds) but I am ok with that since its only an internal package to move bits between CoreFx and Core-Setup.